hooks/tk-multi-publish2/basic/publish_movie.py

```diff
@@ -473,8 +737,144 @@ def _unreal_render_sequence_to_movie(self, destination_path, unreal_map_path, se
         cmdline_args.append("-NoScreenMessages")
 
         unreal.log("Sequencer command-line arguments: {}".format(cmdline_args))
-        
+
         # Send the arguments as a single string because some arguments could contain spaces and we don't want those to be quoted
         subprocess.call(" ".join(cmdline_args))
 
-        return os.path.isfile(output_filepath), output_filepath
+        return os.path.isfile(output_path), output_path
+
+    def _unreal_render_sequence_with_movie_queue(self, output_path, unreal_map_path, sequence_path, presets=None):
+        """
+        Renders a given sequence in a given level with the Movie Render queue.
+
+        :param str output_path: Full path to the movie to render.
+        :param str unreal_map_path: Path of the Unreal map in which to run the sequence.
+        :param str sequence_path: Content Browser path of sequence to render.
+        :param presets: Optional :class:`unreal.MoviePipelineMasterConfig` instance to use for renderig.
+        :returns: True if a movie file was generated, False otherwise
+                  string representing the path of the generated movie file
+        """
+        output_folder, output_file = os.path.split(output_path)
+        movie_name = os.path.splitext(output_file)[0]
+
+        qsub = unreal.MoviePipelineQueueEngineSubsystem()
+        queue = qsub.get_queue()
+        job = queue.allocate_new_job(unreal.MoviePipelineExecutorJob)
+        job.sequence = unreal.SoftObjectPath(sequence_path)
+        job.map = unreal.SoftObjectPath(unreal_map_path)
+        # Set settings from presets, if any
+        if presets:
+            job.set_preset_origin(presets)
+        # Ensure the settings we need are set.
+        config = job.get_configuration()
+        # https://docs.unrealengine.com/4.26/en-US/PythonAPI/class/MoviePipelineOutputSetting.html?highlight=setting#unreal.MoviePipelineOutputSetting
+        output_setting = config.find_or_add_setting_by_class(unreal.MoviePipelineOutputSetting)
+        output_setting.output_directory = unreal.DirectoryPath(output_folder)
+        output_setting.output_resolution = unreal.IntPoint(1280, 720)
+        output_setting.file_name_format = movie_name
+        output_setting.override_existing_output = True  # Overwrite existing files
+        # Remove problematic settings
+        for setting, reason in self._check_render_settings(config):
+            self.logger.warning("Disabling %s: %s." % (setting.get_name(), reason))
+            config.remove_setting(setting)
+
+        # Default rendering
+        config.find_or_add_setting_by_class(unreal.MoviePipelineDeferredPassBase)
+        # Render to a movie
+        config.find_or_add_setting_by_class(unreal.MoviePipelineAppleProResOutput)
```

Comment:
Unless you want to go down frame-based sequences, this is probably the most reasonable direct output format, despite its size.